os/drivers/lcd/Make.defs

```diff
@@ -89,6 +89,9 @@ ifeq ($(CONFIG_LCD),y)
 # Support for the generic LCD framebufer front-end
 CSRCS += lcd_dev.c
 
+# Support for LCD initial bootup screen
+CSRCS += lcd_logo.c
```

Comment:
Don't you need to cover this with the conditional?
Is this working well without CONFIG_LCD_LOGO enabled?